README.md

```diff
@@ -11,7 +11,7 @@ Note that druid syntax is defined in consul (and druids are issued by the SURI s
 
 Druid format:
 
-    bbdddbbdddd (two letters two digits two letters 4 digits)
+    bbddbbdddd (two letters two digits two letters 4 digits)
```

Comment:
i think this should say
` two letters three digits two letters 4 digits `

Reply:
of course!